client/src/js/collections/DogsCollection.js

```diff
@@ -1,14 +1,67 @@
+var $ = require('jquery');
 var Backbone = require('backbone');
 
+var statusMap = {
+  1: {
+    color: 'green',
+    name: 'in office'
+  },
+
+  2: {
+    color: 'yellow',
+    name: 'on a walk'
+  },
+
+  3: {
+    color: 'red',
+    name: 'missing'
+  }
+};
+
 var Dog = Backbone.Model.extend({
   defaults: {
     display_id: 0,
     name: '',
     short_description: '',
     checked_in: false,
-    avatar: ''
+    avatar: '',
+    current_status: 1,
+    current_status_verbose: 'in office',
+    current_status_color: 'green'
```

Comment:
Re: the last bullet, that function should live in the [DashboardDogsListDogView](https://github.com/istrategylabs/paw/blob/master/client/src/js/views/DashboardDogsListDogView.js) because it would be derived from the Dog's status and is not business logic.